koji/katello-mash-split.py

```diff
@@ -317,14 +317,18 @@ def main():
     mash_config = "katello-{}-rhel7".format(version)
     if version == 'nightly':
         mash_config_candlepin = "katello-thirdparty-candlepin-rhel7"
+        mash_config_pulp = "katello-nightly-pulpcore-el7"
```

Comment:
Isn't it sufficient to do this outside of the if? This is equivalent to `"katello-{}-pulpcore-el7".format(version)`

Reply:
Yup, at the time, it felt more consistent for some reason.  But definitely can move it up.

Also, something I just thought about, we aren't preventing this on any versions outside of nightly at the moment... so it may crash older katello mashes